include/git2/remote.h

```diff
@@ -41,6 +41,52 @@ GIT_EXTERN(int) git_remote_create(
 		const char *name,
 		const char *url);
 
+/**
+ * Remote creation options structure
+ *
+ * Initialize with `GIT_REMOTE_CREATE_OPTIONS_INIT`. Alternatively, you can
+ * use `git_remote_create_init_options`.
+ *
+ */
+typedef struct git_remote_create_options {
+	unsigned int version;
+
+	/** The repository that should own the remote.
```

Comment:
Our struct member doc comment style is to either have a single `/** foobar */` line or start with an empty `/**` line.